canonical/cloud-init
# Summary: Add 'network-config' support in ovf-env.xml through VMware g…

…uestinfo.ovfEnv

Details:
1. Support guest set network config through guestinfo.ovfEnv using OVF
2. 'network-config' Property is optional
3. 'network-config' Property's value has to be base64 encoded

Added unittests and updated ovf-env.xml example

## Test Steps
1. Create a ovf-env.xml which includes 'network-config' Property
2. Set 'network-config' Property's value to base64 encoded network config
3. Load ovf-env.xml to VMware guestinfo.ovfEnv
4. Verify cloud-init can get network config from guestinfo.ovfEnv in OVF

## Checklist:
